book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md

```diff
@@ -0,0 +1,83 @@
+- Reviewer:
+- Topics: Units of Measurement, Controlled Vocabulary, Validation in Excel
+- Format: Markdown file
+- Scenarios: Working up data for publication
+- Skills: You should be familiar with
+    - [SI Units of Measurement](https://www.bipm.org/en/measurement-units)
+    - [Controlled vocabularies](https://en.wikipedia.org/wiki/Controlled_vocabulary)
+    - [Functions in Microsoft Excel](https://support.microsoft.com/en-au/office/excel-functions-by-category-5f91f4e9-7b42-46d2-9bd1-63f26a86c0eb)
+- Learning outcomes: After reading about this technique you should:
+    - Understand how to digitally annotate data with units.
+    - Appreciate how storing units digitally enables interoperability and promotes data reuse.
+    - Using Excel (and CSV) as a supplemental information file format.
+- Citation: 
+- Reuse: This notebook is made available under a [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/) license.
+```
+
+## Improving Supplemental Information
+It has become obvious that publishing data with a research paper should no longer be done using the PDF file format.
+Conversion of data from Excel, Word, SASS, etc., to PDF makes it almost impossible to extract the data and in addition
+understand what data values are in terms of the quantity and unit.  To enable interoperability in science we must
+publish our data in a way that makes it easy to accurately interpret and subsequently reuse, while making this process
+easy to implement in research workflows. This technique describes is a low barrier way to improve the quality of data 
+reported as supplemental information and for research data archiving in general.
+
+## Improving data quality through standardized unit representation
+Most scientists use the International System of Units (SI) and for the most part it is the 'easiest' part of doing research.
+However, because it is easy it is often not appreciated how important it is.  In the context of digital data, it is
+especially important for computers to be able to interpret data, especially if we expect them to do "calculations for us" 
+in any accurate and reliable way. {cite:p}`Hanisch2022`
+
+There are many ways in which researchers represent units as they work up their data.  It can be part of the header of
+a column, appended to numeric values as a string, or in some situations implied by the quantity being measured.  The
+common thread here is that units are not usually explicitly (separately) reported and that makes them difficult to deal
+with digitally.  In addition, in many situations a unit is not written in accordance to published guidelines, which in 
+chemistry is the IUPAC "Quantities, Units and Symbols in Physical Chemistry" (or The IUPAC Green Book). {cite:p}`Green2007`
+
+There is a lot of information in the exhaustive tables in the Green Book, and it covers, not surprisingly, a wide 
+range of quantities and units across the many subdisciplines of chemistry. As a guide for humans to read it fine but
```

Comment:
it's

Reply:
Thanks, will fix.

Reply:
OK fixed in commit 900659894500da82eb08d47a8eb48e7f752f7c15.  Commit message is not what I entered in PyCharm... :(